### DISCUSSION TITLE
Astrotomic/laravel-translatable

### DISCUSSION THREAD

**Discussion by Dr-Maspix** [filamentphp/filament#4099] [Help] [answered]
Dears, 

is there is any one do full implementation for this package ,
as per i have issue i need to make two inputs for name for example ,
and i need two inputs one for en and one for ar , 
and i need two inputs in one form and after submit two langs inserted into DB , 

Regards

**Comment by Oleksandr-Moik** [comment, marked as answer]
Hi! It was a long time, but I think a solution has appeared - https://github.com/CactusGalaxy/FilamentAstrotomic

Examples of usage are here: https://github.com/CactusGalaxy/GalaxyStoreExample
